docs/account-portal/overview.mdx

```diff
@@ -4,62 +4,35 @@ description: The Account Portal offers a comprehensive solution for managing use
 ---
 
 import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+import { Tab, Tabs } from 'nextra-theme-docs';
 
-
-# Introduction
-
-The Account Portal offers a comprehensive solution for managing user authentication and profile management in your web application and is the fastest way to add Clerk's authentication to your application with minimal code required. 
+The Account Portal in Clerk is a powerful feature that allows you to streamline the sign-in, sign-up, and user profile management experience for your users, all while maintaining seamless integration with your application without having to build your own components or host your own pages.
 
 ![Account Portal](/docs/images/account-portal/account_portal_splash.png)
 
-# Getting started
-
-To use the Account Portal, add Clerk to your application and protect your pages. Any time an unauthenticated user attempts to access a protected page in your application, they will automatically get redirected to the Account Portal pages to Sign in.
-
-Depending on your choice of framework, you can redirect to them from your application using our NextJS [authMiddleware](/docs/references/nextjs/auth-middleware), React [Control Components](/docs/components/overview) or [ClerkJS SDK](/docs/references/javascript/overview).
-
-
-# Features overview
-
-## Dedicated hosted pages
-
-For every instance, Clerk offers sign-up, sign-in, and user profile pages out of the box. Additionally, if you are using our Organizations feature, you will also get the organization profile page 
-
-For development instances, Clerk will issue you a domain on "lcl.dev". In production, by default, the URLs for your Account Portal are the following: 
-
-```sh 
-https://accounts.[your-domain].com/sign-in
-https://accounts.[your-domain].com/sign-up
-https://accounts.[your-domain].com/user
-https://accounts.[your-domain].com/organization
-https://accounts.[your-domain].com/create-organization
-``` 
-
-
-<Callout type="info" emoji="ℹ️">
-Note that when you go to production, this is why Clerk requires adding the `accounts` CNAME record as part of your DNS settings. If you do not want to do this and are using your own components, you may disable the Account Portal pages via the 'Danger' tab.
-</Callout>
-
-## Custom redirects
-
 
-With the Account Portal, we default the user back to where they came from after a successful auth event (sign in or sign up) and pass this parameter via a `redirect_url` query param.
+# Why use the Account Portal?
 
-For situations where this may not be possible, Clerk defaults to your application homepage and expects the appropriate redirects will kick in. You can also specify default fallbacks directly in the dashboard as well. 
+The Account Portal provides all pages out of the box hosted on Clerk servers for you and requires minimal setup to get started, if you’re looking for the fastest way to get your application up and running, then this is a great choice. 
```

Comment:
This phrasing is very confusing. "provides all pages out of the box" what pages?